### DISCUSSION TITLE
小米WR30U AX3000 112m-nmbm 大分区的固件新版本不见了？

### DISCUSSION THREAD

**Discussion by upsangel** [immortalwrt/immortalwrt#1653] [Q&A]
从 https://downloads.immortalwrt.org/releases/23.05.2/targets/mediatek/filogic/ 版本开始玩，最近想升新 24的RC3，但是发现 最新的 (https://downloads.immortalwrt.org/releases/24.10.0-rc3/targets/mediatek/filogic/) 中不见了112m-nmbm 大分区的固件。 对照 Openwrt 官线也是同样情况，请问是有什么故事或原因吗？

**Comment by Zhang-Siyang** [comment]
同问，楼主解决了吗？

**Comment by 1715173329** [comment]
Duplicate of #1716
